src/main/java/org/myrobotlab/service/data/TopicChange.java

```diff
@@ -0,0 +1,21 @@
+package org.myrobotlab.service.data;
+
+public class TopicChange {
```

Comment:
Would like javadoc on this class and all its members

Reply:
fixed